python/openvino_tokenizers/utils.py

```diff
@@ -222,12 +222,48 @@ def get_hf_tokenizer_attribute(
 ) -> Any:
     return next((value for attr in attributes if (value := getattr(hf_tokenizer, attr, None)) is not None), None)
 
+def update_rt_info_with_versions(ov_tokenizer: Model) -> None:
+    """Updates the runtime information of the OpenVINO tokenizer model with the versions of the 
+    Huggingface tokenizer, Tiktoken, Sentencepiece and OpenVINO runtime.
 
-def update_rt_info(
+    :param ov_tokenizer: Thes OpenVINO tokenizer model to update.
+    :type ov_tokenizer: openvino.Model
+
+    """
+    def get_package_version(name: str) -> str:
+        import importlib.metadata as metadata
+        try:
+            return metadata.version(name)
+        except metadata.PackageNotFoundError:
+            return None
```

Comment:
No need for a nested local function here, it can be declared outside.

Reply:
done